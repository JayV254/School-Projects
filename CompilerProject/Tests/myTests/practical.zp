int main()
{
    int i, j, rows;

    puts("Enter number of rows: ");
    rows = getnum();

    for(i=1; i<rows+1; i=i+1)
    {
        for(j=1; j<i+1; j=j+1)
        {
            puts("* ");
        }
        puts("\n");
    }
    return 0;
}
