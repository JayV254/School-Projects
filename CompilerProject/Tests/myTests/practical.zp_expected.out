Enter number of rows: input-> 5
*
* *
* * *
* * * *
* * * * *

